mri/operators/fourier/non_cartesian.py

```diff
@@ -445,6 +445,13 @@ def __init__(self, samples, shape, n_coils=1, density_comp=None,
             self.samples = samples
         if density_comp is None:
             density_comp = np.ones(samples.shape[0])
+        if smaps is None:
```

Comment:
wasn't this needed before because you didn't test or am I getting something wrong?

Reply:
We internally in C code just passed Nine by default earlier. It is only now that we have started supporting this

Reply:
ok makes sense